forui/test/src/widgets/line_calendar_golden_test.dart

```diff
@@ -0,0 +1,110 @@
+@Tags(['golden'])
+library;
+
+import 'package:flutter/widgets.dart';
+
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:forui/forui.dart';
+import 'package:forui/src/widgets/line_calendar/line_calendar.dart';
+import '../test_scaffold.dart';
+
+void main() {
+  group('FLineCalendar', () {
+    testWidgets('blue screen', (tester) async {
+      await tester.pumpWidget(
+        TestScaffold.blue(
+          child: Center(
+            child: Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 20),
+              child: FLineCalendar(
+                style: TestScaffold.blueScreen.lineCalendarStyle,
+                controller: FCalendarController.date(),
+              ),
+            ),
+          ),
+        ),
+      );
+
+      await expectLater(find.byType(TestScaffold), isBlueScreen);
+    });
+
+    for (final theme in TestScaffold.themes) {
+      for (final (lineCalendar, controller) in [
+        ('default', FCalendarController.date(initialSelection: DateTime.utc(2024, 10, 20))),
+      ]) {
```

Comment:
Is this for-loop necessary?

Reply:
I thought it might be useful if we end up having to adjust LineCalendar in the future to support other controller types

Reply:
I think we could remove it for now and re-introduce if we ever decide to down the line.